TPU profiling extension

ketyi opened the pull request:
TensorBoard startup is extended with TPU profiling option (in case of TPU is present in the Colab environment)

See: https://cloud.google.com/tpu/docs/cloud-tpu-tools

ketyi committed:
TensorBoard startup is extended with TPU profiling option (in case of TPU is present in the Colab environment)

taomanwai reviewed (commented):
Thanks for the contribution!

But when I tried, error "No OpKernel was registered to support Op 'ConfigureDistributedTPU' with these attrs" occurs!

Do u know why? Or do u have full set of testing code which works? So that i can debug too!

ketyi committed:
Using Tensorflow's Keras

To avoid the "No OpKernel was registered to support Op 'ConfigureDistributedTPU' with these attrs"

ketyi committed:
Using Tensorflow's Keras

To avoid the "No OpKernel was registered to support Op 'ConfigureDistributedTPU' with these attrs"

ketyi commented:
I think this TPU error has nothing to do with my modifications. In my experience it can be due to compatibility issues between the installed Tensorflow and Keras versions. A solution is to import the Keras pieces from tensorflow.python.keras . I have modified it accordingly, could you give it a try?
